### ISSUE TITLE
Defense Grid 2 no longer playable. Image messed up in HMD

### ISSUE THREAD

**Issue by Kiefy** [LibreVR/Revive#1161]
### Description of the issue
  - Game loads fine, the main menu is extremely fov warped, but when you enter the game/tutorial, the image in HMD is completely messed up and unplayable. as if the screen is hyper zoomed in.

### Technical Information
  - CPU: 8700k
  - GPU: GTX 1080
  - RAM: 16GB DDR3200
  - OS: Windows 10 Pro x64
  - HMD: Vive

#### Versions and features
  - Revive: Injector and Patcher
  - SteamVR: beta 1.2.6
  - Revive installation version: 1.6.2

#### What is the last working version before this issue first came up?
  - Revive installation: Not sure. I am only just attempting to play for the first time. however, people have played successfully in the past, as is evident by https://github.com/LibreVR/Revive/issues/805
  - SteamVR: See above

### What apps/games are you trying to run?
  - Defense Grid 2 - Enhanced VR Edition

## How to reproduce your issue?
  1. Launch the game with Patch or Inject

**Comment by elnarvideo** [comment]
Yup. Getting the same thing in most Oculus games but in my case its in Remixed, not Revive. Im using Samsung Odyssey+

**Comment by swiznoob** [comment]
Same thing, still not fixed. Worked fine last year when I played. Severe main menu fov warping when moving the head. Starting any mission, image hyper zoomed, unplayable. Bringing up the menu during a mission, hyper zoom goes away- back to the crazy fov warping when moving the head, but resuming the mission causes hyper zoom again.

CPU: i7-4790k
GPU: GTX 1080ti
RAM: 24GB
OS: Windows 10 Pro x64
HMD: Vive Pro

SteamVR: 1.2.10
Revive: 1.6.3.0

Tried launching the game through both the Injector and Revive Dashboard, same result on both.

**Comment by LuckMic** [comment]
Is there a chane to get a fix for the games?

**Comment by jwwaldre** [comment]
Has anyone managed to get Defense Grid 2 to work since the problem was first announced?  Latest version shows same issue for me.  Unplayable. :(

**Comment by ozdeadmeat** [comment]
Late 2022, still no fix. :(

Would be nice if Meta stopped us from needing to use there shit hardware.
